### ISSUE TITLE
Add a community tab to the statistics page

### ISSUE THREAD

**Issue by malachig** [griffithlab/civic-client#788]
It would be good to have automatically generated statistics on the community for various promotion activities:

It could live here:
https://civic.genome.wustl.edu/statistics/community

Lots of things that could be summarized but for starters:
- Summarize the number of users that have earned various badges
- Summarize the number of registered users by role (admin, editor, curator)
- Summarize the size of defined organizations
- Summarize users by their country of origin
- Distribution of all time activity count for all users
- Summarize users by the area of expertise (research scientist, clinician scientist, etc.)

**Comment by susannasiebert** [comment]
@malachig I don't think we currently capture a user's country of origin. Did you have a specific method in mind as to how to determine this? We could add the country for each organization or to the user profile. The first would give us some immediate results, the latter would depend on users updating their profile. We could also attempt to parse their email address but that would probably give us inaccurate results.

**Comment by ahwagner** [comment]
We have 88 contributors total. If we grab their email addresses, we could manually assign their country of origin if their address is affiliated with a university or institute based in a single country.

**Comment by susannasiebert** [comment]
I have a work-in-progress PR for this issue https://github.com/griffithlab/civic-server/pull/354. It's missing "Distribution of all time activity count for all users."

**Comment by susannasiebert** [comment]
griffithlab/civic-server#354 has been merged in. The new endpoint can be found at `api/stats/community`. Once this is deployed into production we can manually set our users' country based on their email address.

**Comment by kkrysiak** [comment]
I can add a country in the admin interface, but despite the fact that it says successfully updated, the country on my admin profile remains empty.

**Comment by lsheta** [comment]
We could integrate information from google analytics, which shows country, number of user, most visited pages, etc.

**Comment by ShahilPema** [comment]
The reporting API should allow integration into the stats page. I added a custom dashboard titled "geography" under the customization --> dashboard tab on the left. All of this is a little over my head, but the links below might be useful:
https://developers.google.com/analytics/devguides/reporting/core/v4/
https://developers.google.com/analytics/devguides/reporting/embed/v1/getting-started

We may need a service account to bypass the log-in: 
https://developers.google.com/analytics/devguides/reporting/core/v3/quickstart/service-php
...but this may also allow access to the rest of the analytics data which maybe shouldn't be public. Again, not quite sure how all of this works.
